### PR TITLE
Add DeSci

### PR DESCRIPTION
**Pull request by gingerheart86**
Hello! 
Just added the awesome-desci repo on your list. 

 

**[https://github.com/DeSciWorldDAO/awesome-desci#readme]**

**[(De)centralized (Sci)ence is a nascent movement in the web3 space, where scientists are looking for blockchain-based solutions to structural issues we've got in traditional science. We are a community project that aims to be the global hub for the DeSci community. We would be very thankful if you add our list to your repo!
]**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

unicorn

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `add Swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The title of your entry should be title-cased and the URL to your list should end in `#readme`.
	- Example: `- [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.`
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. **Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for incubation visibility**.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

### CODE REVIEW

**Comment by Jogai** [comment]
Rules 4, 5 & 7 are not followed, which makes me think you didnt read 1 & 3 closely neither...

**Comment by gingerheart86** [comment]
Hey Jogai
I have read. 
Maybe I didn't understand.
Those rules you mention, do they have numbers? What are you referring to?

I'm sorry for the inconvenience.
I'm quite new to Github, just learning.
Thanks!

**Comment by Jogai** [comment]
Hey, no problem, and welcome to github! Its not that you are wasting my time, I'm just a bystander basically, its the maintainers' time. You can imagine when there are lots of submissions that it takes a lot of time.

**Comment by gingerheart86** [comment]
Yes! I do.
Well, I think I now integrated he requested features.
Would you please take a look?

**Comment by Jogai** [comment]
> title in the format `Add Name of List`

>  the URL to your list should end in `#readme`

**Comment by gingerheart86** [comment]
Hey Jogai.

I corrected most errors from the lint, but I don't know they are still showing,
One is related to the badge. I don't understand since I copied the link badge from other listed list.
Also, I made the changes you asked for.
Would you let me know?

Thank you!

**Comment by gingerheart86** [comment]
Hey Jogai,
All done.
I don't understand which is the problem related to the header.
Thanks

**Comment by tomByrer** [comment]
> * **Has been around for at least 30 days.**

FAIL: 1st commit is Nov 18, 2022
not that big yet either IMHO

**Comment by gingerheart86** [comment]
> > * **Has been around for at least 30 days.**
> 
> FAIL: 1st commit is Nov 18, 2022 not that big yet either IMHO

Hey,
Thanksg for your feedback. 
It's not that big because the space is very new.
But is one of the most expanding spaces in web3, that's why is awesome and it deserves a list.

File: readme.md [hunk 1]
```diff
@@ -807,6 +807,7 @@
 - [Humane Technology](https://github.com/humanetech-community/awesome-humane-tech#readme) - Open source projects that help improve society.
 - [Mental Health](https://github.com/dreamingechoes/awesome-mental-health#readme) - Mental health awareness and self-care in the software industry.
 - [Neuroscience](https://github.com/analyticalmonk/awesome-neuroscience#readme) - Study of the nervous system and brain.
+- [Decentralized Science](https://github.com/DeSciWorldDAO/awesome-desci#readme) - Descentralized Science: Science using blockchain technology to solve issues related to funding, publishing, data validation and sharing.
```
Comment (ademidun) [inline readme.md:810]: Looks good @gingerheart86! I'm a big DeSci fan myself, thanks for adding this!

 Just a few small comments:
The link text already says Decentralized Science, you don't need to mention it again in the description. I also made the description a bit more gramatically correct and concise.

```suggestion
- [Decentralized Science](https://github.com/DeSciWorldDAO/awesome-desci#readme) - Use blockchain technology to develop, fund and share science.
```

**Review by ademidun — APPROVED** [review]
Looks good! The changes I've suggested are recommended but  non-blocking.

File: readme.md (hunk 1 above)
Comment (gingerheart86) [reply readme.md:810]: hey! I liked your suggestion! thanks! I'm not native english speaker so grammar corrections are most welcome!
I think I might change blockchain for DLT

File: readme.md [hunk 2]
```diff
@@ -807,7 +807,7 @@
 - [Humane Technology](https://github.com/humanetech-community/awesome-humane-tech#readme) - Open source projects that help improve society.
 - [Mental Health](https://github.com/dreamingechoes/awesome-mental-health#readme) - Mental health awareness and self-care in the software industry.
 - [Neuroscience](https://github.com/analyticalmonk/awesome-neuroscience#readme) - Study of the nervous system and brain.
-- [Decentralized Science](https://github.com/DeSciWorldDAO/awesome-desci#readme) - Use blockchain technology to develop, fund and share science.
+- [Decentralized Science](https://github.com/DeSciWorldDAO/awesome-desci#readme) - Distributed Ledger Technology (DLT) to develop, fund and share science.
```
[the remark this answers is not in the document]
Comment (gingerheart86) [reply readme.md:810]: I know, but it's true that DLT is more broad. We may be losing concept by just including blockchain.
Either way, I see your point.
We can leave it that way for now.

**Comment by gingerheart86** [comment]
Hey @ademidun!  why the job is still failing?

**Comment by gingerheart86** [comment]
Hey @Zackaryjacob would you mind review it again? Thanks!

**Review by waleedsadek-panx — CHANGES_REQUESTED** [review]
Hello Gingerheart,

I have ran `awesome-lint` on your project and found the below 3 issues:

1.   ✖   1:1   Missing content headers                               remark-lint:awesome-toc
2.   ✖  85:23  https://desci.world/project/635b53500b57f22ac9a041d7  remark-lint:double-link
3.   ✖  90:14  https://desci.world/project/635b53500b57f22ac9a041d7  remark-lint:double-link

Make sure to have `Contents` as mentioned in the guide and remove the duplicate link.

**Review by andrzejsydor — APPROVED** [review]

**Comment by gingerheart86** [comment]
Hey @waleedsadek-panx 
I corrected the duplicates.
The list does have a contents header! why is that error still showing up?
Thanks!

**Review by waleedsadek-panx — CHANGES_REQUESTED** [review]
So according to awesome-lint, in your `readme.md` file, you need to remove extra space before `# Awesome DeSci`  on line 1 and ` ## Contents` on line 7. 

The first two issues are due to this link ` 📖[DeScitionary](#desci-dictionary)` it doesn't work since it's still work in progress, you can remove it from the `Contents` and added later once it's done.

**Comment by gingerheart86** [comment]
@waleedsadek-panx oh!
Just corrected those. 
Now?

**Review by waleedsadek-panx — CHANGES_REQUESTED** [review]
Yes, if you run `awesome-lint` you'll see you're now getting 3 errors instead of 4 errors. You still need to remove the indentation/space before the `## Contents` section on line 7 still. Make sure to run `npx awesome-lint https://github.com/DeSciWorldDAO/awesome-desci` from your terminal to make sure it's passing.

**Review by sophacola — APPROVED** [review]

**Review by gingerheart86 — COMMENTED** [review]
Hi,
I run  `awesome-lint` from my end and it's ok!

**Review by sophacola — APPROVED** [review]

**Review by sophacola — APPROVED** [review]

**Comment by gingerheart86** [comment]
Lint OK.
I put the list under decentralized systems, I think it's a more appropriate classification.

**Review by sophacola — APPROVED** [review]

**Review by sophacola — APPROVED** [review]

**Review by waleedsadek-panx — APPROVED** [review]
It's linting all is good! The only thing left I can see now is changing the commit title from `Add Awesome DeSci` to `Add DeSci` and removing the extra line you've added on line 812.

**Comment by gingerheart86** [comment]
Done! 
Thank you :)

> It's linting all is good! The only thing left I can see now is changing the commit title from `Add Awesome DeSci` to `Add DeSci` and removing the extra line you've added on line 811.

**Review by sophacola — APPROVED** [review]

File: readme.md [hunk 3]
```diff
@@ -808,6 +809,7 @@
 - [Mental Health](https://github.com/dreamingechoes/awesome-mental-health#readme) - Mental health awareness and self-care in the software industry.
 - [Neuroscience](https://github.com/analyticalmonk/awesome-neuroscience#readme) - Study of the nervous system and brain.
```
Comment (sophacola) [inline readme.md:811]: Desci

**Review by sophacola — APPROVED** [review]

**Comment by ademidun** [comment]
> Lint OK.
> I put the list under decentralized systems, I think it's a more appropriate classification.

@gingerheart86 why did you change it?

I think the previous section you had originally was more appropriate. We have to think about who is more likely to be interested in DeSci and who do we want to attract. People interested in science and health or people interested in decentralized systems?

If a decentralized systems person sees desci but is not interested in science, they won’t get much value.

While a science person would find desci useful even if they’re not interested in decentralization. Because an interest in decentralization is not a prerequisite for finding value in desci, all you need is an interest in better ways to do science.

**Comment by gingerheart86** [comment]
Hey @ademidun 
I see your point and it's valid.
I do believe on the contrary that is it's through decentralization (not necessarily on chain) that we will get over the pain points of traditional science. 
I changed it because DeSci as you said is focused on get science better. And Science goes far beyond health and social sciences. Under that classification, you don't take the hard sciences and other natural sciences which are not related to human health into account. Although it's true that most DeSci projects now are from the latter, we are just getting started and the idea is to cover all scientific disciplines. 

However I don't see any problem to keep it under the former classification for now. 

> > Lint OK.
> >  @I put the list under decentralized systems, I think it's a more appropriate classification.
> 
> @gingerheart86
> 
> I think the previous section you had originally was more appropriate. We have to think about who is more likely to be interested in DeSci and who do we want to attract. People interested in science and health or people interested in decentralized systems?
> 
> If a decentralized systems person sees desci but is not interested in science, they won’t get much value.
> 
> While a science person would find desci useful even if they’re not interested in decentralization. Because an interest in decentralization is not a prerequisite for finding value in desci, all you need is an interest in better ways to do science.

**Comment by gingerheart86** [comment]
Hey @ademidun what should I do then? Is it mandatory to change it back or do you accept the changes?

**Comment by gingerheart86** [comment]
Thanks @Zackaryjacob!
@sindresorhus  Would you merge the pull request?

**Comment by sindresorhus** [comment]
The formatting for the "Scientific papers" is a bit messy. The title should come first. It should be a proper list using `- `. A short summary of each paper would also be useful.

---

In the "Articles" section, the title should come first.

---

The items in the "Blogs" section need a description.

---

> Every Cure For Every Community- The Podcast - VibeBio.

These entries need better description. "VibeBio" says nothing to me.

---

Awesome lists are English. Resources for other languages should be a separate Markdown file.

---

https://github.com/DeSciWorldDAO/awesome-desci#channels

Needs descriptions.

---

The repo and heading of the list does not match the title used in the entry here.

**Comment by gingerheart86** [comment]
Hey! @sindresorhus 

1. The formatting for the "Scientific papers" is a bit messy. The title should come first. It should be a proper list using - . A short summary of each paper would also be useful.  _Formatting follows usual scientific citation, it wouldn't be ok to list them under the title nor giving a summary of each paper. The list resembles a scientific bibliography_
2. In the "Articles" section, the title should come first. Similar to point 1. 
3. In reference to other languages, the idea is to provide a broad list of DeSci content, incluiding content in other languages as it's still very scarse. Would we be able to keep them anyway? we would ppl to know these content exists.
4. I changed the title to "Add DeSci" because I was asked to. Does it need to be changed back to "Add Awesome DeSci"?

about the descriptions you are asking...I think I'm not following you. Isn't kind of obvious that under "Blogs" you can see a list of blogs? same with channels.

Thank you so much for your time
Kind regards
Carolina

**Comment by sindresorhus** [comment]
> Formatting follows usual scientific citation, it wouldn't be ok to list them under the title nor giving a summary of each paper. The list resembles a scientific bibliography

This is not a science paper, it's an Awesome list. It should be user-friendly.

> Would we be able to keep them anyway? we would ppl to know these content exists.

No. This is the same for all Awesome lists. You could create a combined secondary page called `other-languages.md` with content from any other language and link to it from the readme.

**Comment by sindresorhus** [comment]
> about the descriptions you are asking...I think I'm not following you. Isn't kind of obvious that under "Blogs" you can see a list of blogs? same with channels.

Each item in the "blog" section should have a description about what the blog is about.

**Comment by sindresorhus** [comment]
Closing for lack of activity.

**Pull request closed without merging**